### PR TITLE
Library App

### PR DESCRIPTION
**Pull request by Laguna1**
### What this PR does:
Adds functionality to:
- Dynamically create books
- Delete books
- Toggle read-status for books

### CODE REVIEW

File: static/js/main.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  displayBooks();
+}
+
+Book.prototype.toggleReadStatus = function () {
+  this.read = !this.read;
+  displayBooks();
+};
+
+function showForm() { // eslint-disable-line no-unused-vars
+  newBookForm.style.display = 'flex';
```
Comment (lekegitrepo) [inline static/js/main.js]: - **_The styling logic should be in CSS files_**

File: static/js/main.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+function createBook() { // eslint-disable-line no-unused-vars
+  const book = new Book(
+    newBookTitle.value,
+    newBookAuthor.value,
+    newBookPages.value,
+    newBookRead.checked,
+  );
+  addBookToLibrary(book);
+  newBookForm.style.display = 'none';
```
Comment (lekegitrepo) [inline static/js/main.js]: - **_The styling logic should be in CSS files_**

File: static/js/main.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  displayBooks();
+}
+
+function cancelButton() { // eslint-disable-line no-unused-vars
+  newBookForm.style.display = 'none';
```
Comment (lekegitrepo) [inline static/js/main.js]: - **_The styling logic should be in CSS files_**

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
## Status: Request changes :recycle:
#### Hi @pbkabali and @Laguna1 :clap: :clap:
#### The webpage looks great. You have done a good job of implementing a good user interface. However, some requirements are missing/not implemented correctly. Check my comments below :point_down: for the overview of the required changes and the inline comments for the affected part of the code.

##### Project Configuration:
- **_The package.json file is missing which means this is not well configured. You can run `npm init` so that the package.json file will be added._**

##### Dom Manipulation
- **_The application logic should be separated from DOM manipulation tasks ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/dom_logic_separation_requirement.png) from Pathwright)_**

- **_The styling logic should be in CSS files_**

##### Readme:
- **_Add instruction on how to clone and run the project locally._**

#### Optional
- **_Add validations to the form, user would not be able to create an empty book_**

- **_You can add data persistence by saving, retrieving, and updating the library books using [the localstorage API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/localstorage_stretch_requirement.png) from [Odin Project](https://www.theodinproject.com/courses/javascript/lessons/library))_**

- **_Try as much as you can to avoid disabling ESLint rules. However, instead of disabling the same rules in several places, disable it once so that it applies to all the affected parts._**

#### _Reviewed by Temitayo Adeleke_

**Comment by Laguna1** [comment]
Thank you for the review, @lekegitrepo.  We have worked on the feedback and requested another review.

**Review by rOluochKe — APPROVED** [review]
### You got it right, well done. The project has been approved. Congratulations :boom: :100:
### Merge and submit the project. Good luck with the next one.
- Happy coding!

**Pull request merged**